ajax/libs/emojionearea/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "emojionearea", 
+  "description": "EmojiOne Area is a small jQuery plugin that allows you to transform any html element into simple WYSIWYG editor with ability to use Emojione icons. The end result is a secure text/plain in which the image icons will be replaced with their Unicode analogues.",
+  "filename": "emojionearea.min.js",
+  "keywords": [
+	"emoji", 
+	"emojione", 
+	"picker", 
+	"emojionearea"
```

Comment:
Let's remove spare spaces in Line 6-10. We only need two spaces. `keywords` would be
```
"keywords": [
  "emoji",
  "emojione",
  "picker",
  "emojionearea"
],
```